addons/purchase/res_config.py

```diff
@@ -46,6 +46,23 @@ class purchase_config_settings(osv.osv_memory):
                  '-This installs the module stock_dropshipping.'),
     }
 
+    def onchange_costing_method(self, cr, uid, ids, group_costing_method, context=None):
```

Comment:
:no_good:  I really don't like this thing. It breaks the addons orthogonality.

I think this code should lie in stock_landed_costs. By the way you will be able to remove the check on ir.module.module.